{"uid":"8ae7332199b93f8e","name":"test_setup","fullName":"tests.integration.test_smoke#test_setup","historyId":"d623373ae070a62cab8e4b2f9dd522ae","time":{"start":1752456305249,"stop":1752456307267,"duration":2018},"status":"broken","statusMessage":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1670, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 156, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/tests/integration/test_smoke.py\", line 31, in test_setup\n    await ops_test.model.deploy(\"./releases/latest/postgresql-k8s-bundle.yaml\", trust=True)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 1914, in deploy\n    await handler.execute_plan()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 453, in execute_plan\n    await self._resolve_charms()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 436, in _resolve_charms\n    charm_url, charm_origin = await self.model._resolve_charm(\n                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 2078, in _resolve_charm\n    resp = await charms_facade.ResolveCharms(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/facade.py\", line 482, in wrapper\n    reply = await f(*args, **kwargs)\n            ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/_client6.py\", line 276, in ResolveCharms\n    reply = await self.rpc(msg)\n            ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/facade.py\", line 680, in rpc\n    result = await self.connection.rpc(msg, encoder=TypeEncoder)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/connection.py\", line 597, in rpc\n    raise errors.JujuAPIError(result)\njuju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"ops_test","time":{"start":1752456305045,"stop":1752456305248,"duration":203},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1752456305045,"stop":1752456305045,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1752456305045,"stop":1752456305045,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1752456305248,"stop":1752456305248,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1670, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 156, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/tests/integration/test_smoke.py\", line 31, in test_setup\n    await ops_test.model.deploy(\"./releases/latest/postgresql-k8s-bundle.yaml\", trust=True)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 1914, in deploy\n    await handler.execute_plan()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 453, in execute_plan\n    await self._resolve_charms()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 436, in _resolve_charms\n    charm_url, charm_origin = await self.model._resolve_charm(\n                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 2078, in _resolve_charm\n    resp = await charms_facade.ResolveCharms(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/facade.py\", line 482, in wrapper\n    reply = await f(*args, **kwargs)\n            ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/_client6.py\", line 276, in ResolveCharms\n    reply = await self.rpc(msg)\n            ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/facade.py\", line 680, in rpc\n    result = await self.connection.rpc(msg, encoder=TypeEncoder)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/connection.py\", line 597, in rpc\n    raise errors.JujuAPIError(result)\njuju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","steps":[],"attachments":[{"uid":"978b049342fd34c","name":"log","source":"978b049342fd34c.txt","type":"text/plain","size":1095}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"ops_test::finalizer","time":{"start":1752456307286,"stop":1752456307595,"duration":309},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1752456307286,"stop":1752456307286,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1752456307595,"stop":1752456307595,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1752456307596,"stop":1752456307596,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1752456307596,"stop":1752456307596,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_smoke"},{"name":"host","value":"pkrvmq0rgcvqdmg"},{"name":"thread","value":"17020-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_smoke"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"b1f61d339fbfe561","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":0,"broken":52,"skipped":0,"passed":319,"unknown":9,"total":380},"items":[{"uid":"a52a75fb05377121","reportUrl":"../2160//#testresult/a52a75fb05377121","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1752369915417,"stop":1752369917462,"duration":2045}},{"uid":"be2abdd9f0b643b7","reportUrl":"../2159//#testresult/be2abdd9f0b643b7","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1752283534507,"stop":1752283537000,"duration":2493}},{"uid":"8893dc0fef69df7b","reportUrl":"../2158//#testresult/8893dc0fef69df7b","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1752197068369,"stop":1752197070179,"duration":1810}},{"uid":"b4d7dd61817be0bc","reportUrl":"../2157//#testresult/b4d7dd61817be0bc","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1752110640525,"stop":1752110643597,"duration":3072}},{"uid":"9262cb4e33084a1","reportUrl":"../2156//#testresult/9262cb4e33084a1","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1752024273387,"stop":1752024275926,"duration":2539}},{"uid":"a7c806a8db9f0ba2","reportUrl":"../2154//#testresult/a7c806a8db9f0ba2","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751937827669,"stop":1751937831004,"duration":3335}},{"uid":"e14f1cb9d022df23","reportUrl":"../2153//#testresult/e14f1cb9d022df23","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751851469404,"stop":1751851473609,"duration":4205}},{"uid":"40ecfa8f50f8750b","reportUrl":"../2152//#testresult/40ecfa8f50f8750b","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751765084768,"stop":1751765089267,"duration":4499}},{"uid":"7246751a1f41f91b","reportUrl":"../2151//#testresult/7246751a1f41f91b","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751678633268,"stop":1751678636093,"duration":2825}},{"uid":"26bcdfaedacdf47","reportUrl":"../2150//#testresult/26bcdfaedacdf47","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751592250545,"stop":1751592253517,"duration":2972}},{"uid":"fa5a5fd142ade7ea","reportUrl":"../2149//#testresult/fa5a5fd142ade7ea","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751505842363,"stop":1751505843494,"duration":1131}},{"uid":"75fb2a4d0c82bcec","reportUrl":"../2148//#testresult/75fb2a4d0c82bcec","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751419436352,"stop":1751419437078,"duration":726}},{"uid":"d7c1cb3b49ec972a","reportUrl":"../2146//#testresult/d7c1cb3b49ec972a","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751333057688,"stop":1751333060620,"duration":2932}},{"uid":"49e58354ae2d06f2","reportUrl":"../2145//#testresult/49e58354ae2d06f2","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751246702440,"stop":1751246706495,"duration":4055}},{"uid":"e909f1a17f53f05e","reportUrl":"../2144//#testresult/e909f1a17f53f05e","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751160241859,"stop":1751160243729,"duration":1870}},{"uid":"261b1141275ef46f","reportUrl":"../2143//#testresult/261b1141275ef46f","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1751073811350,"stop":1751073812258,"duration":908}},{"uid":"8c0b34bca34adb4b","reportUrl":"../2142//#testresult/8c0b34bca34adb4b","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1750987433053,"stop":1750987435340,"duration":2287}},{"uid":"aff290b2f6983b80","reportUrl":"../2141//#testresult/aff290b2f6983b80","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1750901068453,"stop":1750901069982,"duration":1529}},{"uid":"fea63496ff21bcf7","reportUrl":"../2140//#testresult/fea63496ff21bcf7","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1750814635320,"stop":1750814637893,"duration":2573}},{"uid":"4d3924f96e2c5d0c","reportUrl":"../2138//#testresult/4d3924f96e2c5d0c","status":"broken","statusDetails":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","time":{"start":1750728233801,"stop":1750728234253,"duration":452}}]},"tags":["asyncio","abort_on_fail"]},"source":"8ae7332199b93f8e.json","parameterValues":[]}